spec/service_spec.rb

```diff
@@ -220,6 +220,29 @@ def with_changed_provider_key(from, to)
           s.active?.must_equal false
         end
       end
+
+      describe '.delete_stats' do
+        let(:service_id) { default_service_id }
+        let(:applications) { %w[1 2 3] }
+        let(:metrics) { %w[10 20 30] }
+        let(:users) { %w[100 200 300] }
+        let(:from) { Time.new(2002, 10, 31).to_i }
+        let(:to) { Time.new(2003, 10, 31).to_i }
+        let(:delete_job) do
+          {
+            deletejobdef: {
+              applications: applications,
+              metrics: metrics,
+              users: users,
+              from: from,
+              to: to
+            }
+          }
+        end
+        it 'does not raise error' do
```

Comment:
We should test a case that returns an error like to < from or any other, just to test that the caller gets an error message.